safe/definitions/exposure.py

```diff
@@ -437,12 +441,17 @@
     'size_unit': unit_hectares,
     'units': [unit_hectares],
     'classifications': [
-        generic_landcover_classes, badan_geologi_landcover_classes],
+        generic_landcover_classes,
+        badan_geologi_landcover_classes,
+        data_driven_classes],
     'compulsory_fields': [exposure_type_field],
     'fields': exposure_fields,
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->